tsconfig.json

```diff
@@ -14,7 +14,7 @@
     "moduleResolution": "node",
     "outDir": "../temp/out-tsc",
     "sourceMap": true,
-    "target": "es5",
+    "target": "es6",
```

Comment:
do you need this ? if you already set target: es6 in e2e-bdd/tsconfig.json

Reply:
no, I do not need this change. Thank you, I revert this